tests/suites/1_stateful/04_mini_dataset/04_0001_mini_hits.result

```diff
@@ -0,0 +1,259 @@
+====== SQL31 ======
+2	1124827693	154	77	1750.0
+2	1388696273	50	25	1917.0
+3	1707103414	31	23	1638.0
+2	1320087367	28	14	1368.0
+2	1760151079	28	14	1638.0
+2	-1691584608	28	10	1638.0
+2	-928401671	24	10	1750.0
+2	-1265832712	23	12	1087.0
+2	-1104629711	21	10	1996.0
+2	-430621593	21	12	1368.0
+====== SQL32 ======
+6103609278412364237	-436078919	1	0	1368.0
+4951947865110738245	1873681563	1	1	1368.0
+7060168263352146849	861419998	1	0	1996.0
+7019660611280962560	1119218655	1	1	1750.0
+6637093127069035223	694937018	1	0	1750.0
+5152264301509686058	1752766681	1	1	1638.0
+7806095028878167273	2107471378	1	0	1368.0
+8003672837648309970	2110881137	1	0	1996.0
+5599202363973519738	1530977616	1	1	1368.0
+7054755713861488389	1421038618	1	1	1638.0
+====== SQL33 ======
+6526264117404580809	-83604998	1	1	582.0
+5152264301509686058	1752766681	1	1	1638.0
+7806095028878167273	2107471378	1	0	1368.0
+7947646975100161633	577544894	1	1	1917.0
+7819272473842414756	1355129322	1	1	1638.0
+8291896465054243802	-928854628	1	0	1996.0
+8505348986965736427	682244364	1	0	1996.0
+5915405884132436842	577544894	1	0	1917.0
+7984901677412501734	1744948143	1	1	1638.0
+5439260956883700997	-525132013	1	0	1087.0
+====== SQL34 ======
+https://produkty%2Fproduct	2245
+https://produkty/kurortmag	1154
+https://produkty%2F	1153
+https://produkty%2Fpulove.ru/kiroverlanet.ru/otdam-soul	1020
+https://produkty%2Fpulove.ru/album/login	1003
+https://produkty/full&qset=159508&s_yers	745
+https://produkty%2Fpulove.ru/moscow	525
+https://produkty%2Fpulove.ru/kiroverlanet.ru/otdam-souls 2&where=all&filmId=hzuULzEPUXI&where=all	462
+https://produkty%2Fplata-pr-advert27929217742/4F2E	422
+https://produktov_55_ru	405
+====== SQL35 ======
+1	https://produkty%2Fproduct	2245
+1	https://produkty/kurortmag	1154
+1	https://produkty%2F	1153
+1	https://produkty%2Fpulove.ru/kiroverlanet.ru/otdam-soul	1020
+1	https://produkty%2Fpulove.ru/album/login	1003
+1	https://produkty/full&qset=159508&s_yers	745
+1	https://produkty%2Fpulove.ru/moscow	525
+1	https://produkty%2Fpulove.ru/kiroverlanet.ru/otdam-souls 2&where=all&filmId=hzuULzEPUXI&where=all	462
+1	https://produkty%2Fplata-pr-advert27929217742/4F2E	422
+1	https://produktov_55_ru	405
+====== SQL36 ======
+2145476504	2145476503	2145476502	2145476501	1
+2133342967	2133342966	2133342965	2133342964	1
+2096242558	2096242557	2096242556	2096242555	1
+2037048489	2037048488	2037048487	2037048486	1
+2011901432	2011901431	2011901430	2011901429	1
+1975825145	1975825144	1975825143	1975825142	1
+1974516222	1974516221	1974516220	1974516219	1
+1942435789	1942435788	1942435787	1942435786	1
+1925764365	1925764364	1925764363	1925764362	1
+1904854946	1904854945	1904854944	1904854943	1
+====== SQL37 ======
+====== SQL38 ======
```

Comment:
I believe all queries that produce empty results should be adapted.

Reply:
cc @sundy-li  need help. I use head -n 100000 to get the dataset, empty query data is possible. Should we change data or change sql to fit this? Or just merged it, optimize later.

Reply:
I think it better to adapt the filter of this SQL.

```
SELECT UserID FROM hits WHERE UserID = <a userid in your mini dataset>;
```

Reply:
OK, I will use an exist UserID, how about the other SQLs ?